core/src/main/java/nl/nn/adapterframework/stream/Message.java

```diff
@@ -364,4 +379,17 @@ public static String asString(Object object, String defaultCharset) throws IOExc
 		return Message.asMessage(object).asByteArray(defaultCharset);
 	}
 	
+	private void writeObject(ObjectOutputStream stream) throws IOException {
```

Comment:
Private method that's never been called?

Reply:
Zo werkt dat met Serializable